docs/docs/index.md

```diff
@@ -134,6 +134,38 @@ with q.connection.pipeline() as pipe:
 
 `Queue.prepare_data` accepts all arguments that `Queue.parse_args` does.
 
+### Batching jobs
```

Comment:
I think the phrase `Batching jobs` is slightly misleading. We don't allow jobs to be "batched" [execution](https://en.wikipedia.org/wiki/Batch_processing) wise but just grouped, right? Should we change the implementation to use `Group` instead of `Batch`? I'm also open to other naming ideas that more accurately conveys the use case.

Reply:
@selwin I've moved the enqueueing logic to the batch class to match that interface.

Reply:
I like group. I've changed the name